src/screens/Analytics/SystemMetrics/SystemMetricsAnalyticsUtils.res

```diff
@@ -58,7 +56,14 @@ let defaultColumns: array<
 > = [
   {
     sectionName: "",
-    columns: [Latency, ApiCount],
+    columns: [
+      {
+        colType: Latency,
+      },
+      {
+        colType: ApiCount,
+      },
+    ],
```

Comment:
Can we make a utility func which will generate this object based on the array passed ?